### PR TITLE
products page completed

### PR DESCRIPTION
**Pull request by MayurChawla**
Completed Products page

### CODE REVIEW

File: productsPage/productsPage.html [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+                        <input type="checkbox" value="Women" id="option3">
+                        <label for="option2">3 Stars</label>
+                    </div>
+                    <div class="option2">
+                        <input type="checkbox" value="Women" id="option4">
+                        <label for="option2">2 Stars</label>
+                    </div>
+                    <div class="option2">
+                        <input type="checkbox" value="Women" id="option5">
+                        <label for="option2">1 Stars</label>
```
Comment (omjadhav85) [inline productsPage/productsPage.html]: values should be different here

File: productsPage/productsPage.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+.sidebar-nav .cost-filter .title {
+    padding: 1rem 0;
+}
+.sidebar-nav .cost-filter .slider-container .slider {
+    width: 100%;
+}
+.sidebar-nav .cost-filter .slider-container .slider-values {
+    display: flex;
+    justify-content: space-between;
+}
```
Comment (omjadhav85) [inline productsPage/productsPage.css:49]: Only .slider-container .slider should work I guess

**Review by omjadhav85 — APPROVED** [review]
Some minor changes suggested...else looks good!

File: productsPage/productsPage.css (hunk 2 above)
Comment (MayurChawla) [reply productsPage/productsPage.css:49]: according to me its a good practise to be specific. Correct me or enlighten me please

**Pull request merged**